Pod/Classes/TSSwiftMarkdownParser.swift

```diff
@@ -98,6 +100,19 @@ public class TSSwiftMarkdownParser: TSBaseParser {
         strongAndEmphasisAttributes = [NSFontAttributeName: strongAndEmphasisFont]
         
         if withDefaultParsing {
+            
+            addNumberedListParsingWithMaxLevel(0, leadFormattingBlock: { (attributedString, range, level) in
```

Comment:
Does a max level of zero mean something, or is it just a sentinel indicating there's no max level? If it's the latter it should probably be an optional instead.